logstash-core/spec/logstash/runner_spec.rb

```diff
@@ -33,44 +44,29 @@ def run(args); end
     context "with no arguments" do
       let(:args) { [] }
       it "should show help" do
-        expect(subject).to receive(:show_short_help).once
-        expect(channel).to receive(:fatal).once
         expect(channel).to receive(:warn).once
+        expect(channel).to receive(:fatal).once
+        expect(subject).to receive(:show_short_help).once
         subject.run(args)
       end
     end
   end
 
-  context "when loading the configuration" do
+  context "--agent" do
+    class DummyAgent < LogStash::Agent; end
+
+    let(:agent_name) { "testagent" }
+    let(:dummy_agent_class) { DummyAgentClass }
```

Comment:
I know this is picky, but wording this as dummy it always felt to me strange. Why not using mock or stub? what do you think?